R/plumber.R

```diff
@@ -902,6 +876,54 @@ plumber <- R6Class(
     setErrorHandler = function(fun){
       private$errorHandler <- fun
     },
+    #' @description Add a function to customize what is returned in `$apiSpec()`.
+    #' @param api_fun This function should accept an OpenAPI Specification autogenerated by `plumber`.  It should return a list object in valid OpenAPI Specification format. This will not be validated.
```

Comment:
The docs here feel like they could be fleshed out a little bit.
- What's the exact signature we expect?
- Is there an example input and output?

What it sounds like, and this feels a bit wrong, is that it receives an OpenAPI JSON string and expects a listified version of it back, which sounds a lot like `fromJSON`, with all the weird array encapsulation things implied.

Reply:
> with all the weird array encapsulation things implied

Yup